app/assets/javascripts/discourse/app/initializers/subscribe-user-notifications.js

```diff
@@ -22,38 +22,46 @@ export default {
     const user = container.lookup("current-user:main");
     const bus = container.lookup("service:message-bus");
     const appEvents = container.lookup("service:app-events");
+    const siteSettings = container.lookup("service:site-settings");
 
     if (user) {
       bus.subscribe("/reviewable_counts", (data) => {
-        user.set("reviewable_count", data.reviewable_count);
+        if (data.reviewable_count !== undefined) {
```

Comment:
```suggestion
        if (data.reviewable_count) {
```

I think we can just do this?

Reply:
oh `data.reviewable_count` can be zero in which case we don't the if condition to evaluate to true. I've changed the condition to `>= 0`.

Reply:
O ic. Yea `>= 0` makes it much easier to read. Thank you for the changes :+1: